### ISSUE TITLE
"Done" button unresponsive from Secondary email section

### ISSUE THREAD

**Issue by SorinaFlorean** [mozilla/fxa-content-server#6242]
Environment: 
Platforms: Android, Win10, iOS
Sever: Stage and prod

Steps to reproduce:

1. Login with FxA account and go to manage account page;
2. Tap on "Add" from Secondary email section;
3. Type a new email and choose "Add";
4. Tap on "Done".

Expected results: tapping on "done" button closes the section.
Actual results: nothing happens, the button is unresponsive. 

Video: https://drive.google.com/open?id=1qxDOY7nYPrXm6EiK_L8a4omns3BQ4lpz

**Comment by vladikoff** [comment]
![image](https://user-images.githubusercontent.com/128755/40928653-1d37f762-67f0-11e8-978f-6f6121188e0d.png)

from mtg: fix the css if we can !

**Comment by championshuttler** [comment]
Hi @vladikoff I would like to work on it. Can i take this up?

Thanks

**Comment by DraciVik** [comment]
I would like to try and solve this bug. Can I have a go at it? @vladikoff Thank you!

**Comment by lmorchard** [comment]
@championshuttler, are you still interested in this issue?

@DraciVik, looks like you're taking on Issue #6257 - do you still want to tackle this one?

**Comment by championshuttler** [comment]
@lmorchard yes I am on it :), I am facing some issues while setting up the project in the mean time if Viktor is able to send the PR , feel free to do that.

**Comment by DraciVik** [comment]
@championshuttler @lmorchard. Thank you! I will try and tackle this one if it's not solved when I get to it.

**Comment by karansapolia** [comment]
I would like to tackle this one! I see a lot of interest in this already. If Shivam or Vicktor are not able to give time to this, I would like to work on this next.
